src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve) => {
+  document.querySelector('.logo').addEventListener('click', () => {
+    resolve();
+  });
+});
+
+const promise2 = new Promise((_, reject) => {
+  setTimeout(() => {
+    reject();
+  }, 3000);
+});
+
+const onSuccess = () => {
+  const successMessage = document.createElement('div');
+  successMessage.className = 'message';
+  successMessage.textContent = 'Promise was resolved!';
+  document.body.appendChild(successMessage);
+};
+
+const onError = () => {
+  const errorMessage = document.createElement('div');
+  errorMessage.className = 'message error-message';
+  errorMessage.textContent = 'Promise was rejected!';
+  document.body.appendChild(errorMessage);
+};
+
+promise1
+  .then(onSuccess)
+  .catch(onError);
```

Comment:
The promise handling logic for `promise1` is correct, as it resolves on a click event and calls `onSuccess`. However, the `catch` block will never be executed because `promise1` does not have a rejection path. Consider removing the `catch` block for `promise1` if you don't plan to handle any errors.